bciers/apps/reporting/src/app/components/finalReview/FinalReviewFormNew.tsx

```diff
@@ -0,0 +1,358 @@
+  { label: "Last Name", key: "last_name" },
+  { label: "Job Title / Position", key: "position_title" },
+  { label: "Business Email Address", key: "email" },
+  { label: "Business Telephone Number", key: "phone_number" },
+  { label: "Business Mailing Address", key: "street_address" },
+  { label: "Municipality", key: "municipality" },
+  { label: "Province", key: "province" },
+  { label: "Postal Code", key: "postal_code" },
+];
+
+const additionalDataFields = [
+  { label: "Did you capture emissions?", key: "capture_emissions" },
+  {
+    label: "Emissions (t) captured for on-site use",
+    key: "emissions_on_site_use",
+  },
+  {
+    label: "Emissions (t) captured for on-site sequestration",
+    key: "emissions_on_site_sequestration",
+  },
+  {
+    label: "Emissions (t) captured for off-site transfer",
+    key: "emissions_off_site_transfer",
+  },
+  { heading: "Additional data" },
+  { label: "Electricity Generated", key: "electricity_generated", unit: "GWh" },
+];
+
+const complianceSummaryFields = (products: any[] = []) => [
+  {
+    label: "Emissions attributable for reporting",
+    key: "emissions_attributable_for_reporting",
+    unit: "tCO2e",
+  },
+  {
+    label: "Reporting-only emissions",
+    key: "reporting_only_emissions",
+    unit: "tCO2e",
+  },
+  {
+    label: "Emissions attributable for compliance",
+    key: "emissions_attributable_for_compliance",
+    unit: "tCO2e",
+  },
+  { label: "Emissions limit", key: "emissions_limit", unit: "tCO2e" },
+  { label: "Excess emissions", key: "excess_emissions", unit: "tCO2e" },
+  { label: "Credited emissions", key: "credited_emissions", unit: "tCO2e" },
+
+  { heading: "Regulatory values" },
+  { label: "Reduction factor", key: "regulatory_values.reduction_factor" },
+  { label: "Tightening rate", key: "regulatory_values.tightening_rate" },
+  {
+    label: "Initial compliance period",
+    key: "regulatory_values.initial_compliance_period",
+  },
+  { label: "Compliance period", key: "regulatory_values.compliance_period" },
+
+  ...(products.flatMap((product, index) => [
+    { heading: product.name || `Product ${index + 1}` },
+    {
+      label: "Annual production",
+      key: `products.${index}.annual_production`,
+      unit: "production unit",
+    },
+    {
+      label: "Production data for Apr 1 - Dec 31 2024",
+      key: `products.${index}.apr_dec_production`,
+    },
+    {
+      label: "Production-weighted average emission intensity",
+      key: `products.${index}.emission_intensity`,
+      unit: "tCO2e/production unit",
+    },
+    {
+      label: "Allocated industrial process emissions",
+      key: `products.${index}.allocated_industrial_process_emissions`,
+      unit: "tCO2e",
+    },
+    {
+      label: "Allocated Emissions attributable to compliance",
+      key: `products.${index}.allocated_compliance_emissions`,
+      unit: "tCO2e",
+    },
+  ]) || []),
+];
+
+export const FinalReviewFormNew: React.FC<Props> = ({
```

Comment:
is the plan to replace the pre-existing `FinalReviewForm` component with this `New` component? My concern is the potential for confusion between the two. I'd recommend doing the replacement as part of this PR if possible, or if that's not feasible to create a tech debt ticket to do it later.